Source/EmbeddedBoundary/ParticleScraper.H

```diff
@@ -189,26 +189,26 @@ scrapeParticles (PC& pc, const amrex::Vector<const amrex::MultiFab*>& distance_t
 
                 if (phi_value < 0.0)
                 {
+
                     amrex::RealVect normal = DistanceToEB::interp_normal(i, j, k, W, phi, dxi);
+                    DistanceToEB::normalize(normal);
 
```

Comment:
The code below was extract the **closest** point on the EB, instead of the point of the contact between the trajectory and the EB. Since this is not what we want, we removed that code. The point of contact is now calculated in `ParticleBoundaryBuffer.cpp`, but not here.